src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,21 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDate from;
```

Comment:
Great job going above the minimal requirements and making the from/to in Event of type LocalDate